Competition2017/tests/competition/subsystems/climb/ClimbingSubsystemTest.java

```diff
@@ -0,0 +1,28 @@
+package competition.subsystems.climb;
+
+import org.junit.Test;
+import competition.subsystems.climb.ClimbTestBase;
+
+public class ClimbingSubsystemTest extends ClimbTestBase {
+    
+    @Test
+    public void stopTest(){
+        
+        climb.stop();
```

Comment:
If the command did nothing, you wouldn't know. You can only really test Stop after calling Ascend or Descend.